### PR TITLE
Improve README for Mapzen shutdown and Libpostal service

### PR DESCRIPTION
**Pull request by orangejulius**
Besides a few out of date sections, this PR adds better links for documentation and the new libpostal service. After this is merged we can merge the libpostal service change to staging.

Connects https://github.com/pelias/pelias/issues/703
Connects https://github.com/pelias/api/issues/1072
Connects https://github.com/pelias/dockerfiles/pull/42

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,23 +1,23 @@
 >This repository is part of the [Pelias](https://github.com/pelias/pelias)
->project. Pelias is an open-source, open-data geocoder built by
->[Mapzen](https://www.mapzen.com/) that also powers [Mapzen Search](https://mapzen.com/projects/search). Our
->official user documentation is [here](https://mapzen.com/documentation/search/).
+>project. Pelias is an open-source, open-data geocoder originally built by
```
Comment (missinglink) [inline README.md]: originally sponsored by?

File: README.md [hunk 2]
```diff
@@ -43,40 +43,33 @@ The API recognizes the following properties under the top-level `api` key in you
...
-    "requestTimeout": "1200000",
-    "hosts": [
-      {
-        "protocol": "http",
-        "host": "somesemachine.elb.amazonaws.com",
-        "port": 9200
-      }
-    ]
+    "hosts": [{
+      "host": "elasticsearch"
```
Comment (missinglink) [inline README.md]: this is not totally clear that it refers to a DNS name and also supports an IP address?

File: README.md [hunk 3]
```diff
@@ -43,40 +43,33 @@ The API recognizes the following properties under the top-level `api` key in you
...
     "services": {
-      "pip": {
-        "url": "http://mypipservice.com:3000"
-      },
       "placeholder": {
-        "url": "http://myplaceholderservice.com:5000"
+        "url": "http://placeholder:4100"
+      },
+      "libpostal": {
+        "url": "http://libpostal:8080"
```
Comment (missinglink) [inline README.md]: I still think we should change this to be in the 4000 range, in order to be consistent with the other services :)

Comment (orangejulius) [reply README.md]: We can do that in the dockerfiles repo, but as the [default port for go-whosonfirst-libpostal is 8080](https://github.com/whosonfirst/go-whosonfirst-libpostal#wof-libpostal-server), we can't enforce it everywhere.

File: README.md (hunk 2 above)
Comment (orangejulius) [reply README.md]: Hmm true. The idea was to shorten the config a bit, but maybe this is too short?

**Pull request merged**